tools/tvk-oneclick/tvk-oneclick.sh

```diff
@@ -330,7 +350,8 @@ print(soup.body.find('div', attrs={'class':'yaml-content'}).text)
 sys.stdout.close()
 result = subprocess.check_output("kubectl apply -f license_file1.yaml -n $tvk_ns", shell=True)
 EOF
-  ret=$(kubectl get license -n "$tvk_ns" 2>> >(logit) | grep -q Activated)
+  sleep 5
```

Comment:
Instead of this, add loop with timeout and sleep to check status of license.

Reply:
Added loop